pkg/system/system.go

```diff
@@ -404,6 +404,71 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	util.KubeCreateSkipExisting(sys)
 }
 
+// RunApply runs a CLI command
+func RunApply(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	sys := LoadSystemDefaults()
+	ns := util.KubeObject(bundle.File_deploy_namespace_yaml).(*corev1.Namespace)
+	ns.Name = sys.Namespace
+
+	coreResourcesJSON, _ := cmd.Flags().GetString("core-resources")
+	dbResourcesJSON, _ := cmd.Flags().GetString("db-resources")
+	endpointResourcesJSON, _ := cmd.Flags().GetString("endpoint-resources")
+	useOBCCleanupPolicy, _ := cmd.Flags().GetBool("use-obc-cleanup-policy")
+
+	if useOBCCleanupPolicy {
+		sys.Spec.CleanupPolicy.Confirmation = nbv1.DeleteOBCConfirmation
+	}
+	if coreResourcesJSON != "" {
+		util.Panic(json.Unmarshal([]byte(coreResourcesJSON), &sys.Spec.CoreResources))
+	}
+	if dbResourcesJSON != "" {
+		util.Panic(json.Unmarshal([]byte(dbResourcesJSON), &sys.Spec.DBResources))
+	}
+	if endpointResourcesJSON != "" {
+		if sys.Spec.Endpoints == nil {
+			sys.Spec.Endpoints = &nbv1.EndpointsSpec{
+				MinCount: 1,
+				MaxCount: 1,
+			}
```

Comment:
We cannot assume this, we need to get the actual values that are currently written